src/components/molecules/ChatMessageBox/ChatMessageBox.scss

```diff
@@ -0,0 +1,44 @@
+@import "scss/constants";
+
+.ChatMessageBox {
+  &__form {
+    display: flex;
+    align-items: center;
+    padding-bottom: 5px;
+  }
+
+  &__input {
+    // @debt Workaround to avoid styles set by src/scss/global.scss (fix this when when clean that up)
+    all: unset;
+    outline: none;
+    background-color: $saturated-black;
+    flex: 1;
+    cursor: pointer;
+    font-size: 0.9em;
+    padding: 12px 12px;
+    border-radius: 28px;
+    color: $white;
+  }
+
+  &__submit-button {
+    outline: none;
+    border: none;
+    min-width: 44px;
+    min-height: 44px;
+    border-radius: 50%;
```

Comment:
Can we use one of the border radius variables from `src/scss/constants.scss` here? And if not, can we extract a new one to there, then use that here.